lms/djangoapps/api_manager/courses/views.py

```diff
@@ -927,35 +927,35 @@ class CoursesStaticTabsDetail(SecureAPIView):
     """
     **Use Case**
 
-        CoursesStaticTabsDetail returns a collection of custom pages in the
-        course, including the page content.
+        CoursesStaticTabsDetail returns a custom page in the course,
+        including the page content.
 
     **Example Requests**
 
-          GET /api/courses/{course_id}/static_tabs/{tab_id}
+          GET /api/courses/{course_id}/static_tabs/{tab_url_slug}
+          GET /api/courses/{course_id}/static_tabs/{tab_name}
 
     **Response Values**
 
-        * tabs: The collection of custom pages in the course. Each object in the
-          collection conains the following keys:
+        * tab: A custom page in the course. containing following keys:
 
-          * id: The ID of the custom page.
+          * id: The url_slug of the custom page.
 
           * name: The Display Name of the custom page.
 
           * detail: The content of the custom page as HTML.
     """
 
-    def get(self, request, course_id, tab_id):
+    def get(self, request, course_id, tab_identifier):
```

Comment:
I think we should keep the parameter name `tab_id` as it is used in that way throughout the codebase.